scripts/gsa-approved-software.js

```diff
@@ -42,67 +48,90 @@ const getColor = (status) => {
 const getAttachment = item => ({
   title: `${item.name} - ${item.status}`,
   fallback: `${item.name} - ${item.status}`,
-  fields: [{
-    title: '',
-    value: `this is a ${item.platform} product`,
-    short: false
-  }],
-  color: getColor(item.status),
+  fields: [
+    {
+      title: '',
+      value: `this is a ${item.platform} product`,
+      short: false
+    }
+  ],
+  color: getColor(item.status)
 });
 
-const getMessage = (target) => {
+const getMessage = target => {
   const lookingFor = target.toLowerCase();
   return getCSV().then(body => {
     const matches = [];
 
     return new Promise((resolve, reject) => {
-      csv().fromString(body)
-        .on('csv', (item) => {
+      csv()
+        .fromString(body)
+        .subscribe(item => {
```

Comment:
However, this part is actually new.  `csvtojson` changed its APIs, so `.subscribe` replaced `.on('csv')`, and the whole thing returns a promise now, so no need for the `.done()` event handler - use `.then()` instead.  Finally, the `subscribe()` handler provides a JSON object of the current CSV row (keys are the column names, values are the cell values) whereas the old `csvtojson` provided an array (each corresponded to a column value in the current row).

Reply:
it's weird that `suscribe` is the name of that function given what it does but w/e